airbyte-ci/connectors/ci_credentials/pyproject.toml

```diff
@@ -15,7 +15,11 @@ python = "^3.10"
 requests = "^2.28.2"
 click = "^8.1.3"
 pyyaml = "^6.0"
-common_utils = { path = "../common_utils", develop = true }
+
+# Toggle between these two in order to test 'common_utils' changes locally:
+
+common_utils = { git = "https://github.com/airbytehq/airbyte.git", subdirectory = "airbyte-ci/connectors/common_utils" }
```

Comment:
This will read up whatever's in master at build time, right? Would this break situations where a change in common_utils is in the same pull request as something we're trying to run against?